packages/rs-dpp/src/fee/default_costs/constants.rs

```diff
@@ -1,7 +1,7 @@
 use crate::fee::Credits;
 
 pub const BASE_ST_PROCESSING_FEE: Credits = 10000; // 84000
-pub const FEE_MULTIPLIER: Credits = 2;
+pub const DEFAULT_FEE_MULTIPLIER: Credits = 2;
```

Comment:
We should use already defined type, but it's u16. It's confusing

type FeeMultiplier = u16;

Reply:
No Ivan, this is for something different, this is the default fee_multiplier for the masternode based multiplier.

Reply:
Renaming to DEFAULT_SYSTEM_FEE_MULTIPLIER